src/main/java/uk/gov/hmcts/reform/professionalapi/domain/RefreshUser.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.hmcts.reform.professionalapi.domain;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Setter
+@Getter
+@NoArgsConstructor
+public class RefreshUser {
+
+    private String userIdentifier;
+    private LocalDateTime lastUpdated;
+    private OrganisationInfo organisationInfo;
+    private List<UserAccessType> userAccessTypes;
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    private LocalDateTime dateTimeDeleted;
+
+    public RefreshUser(String userIdentifier,
```

Comment:
**import lombok.AllArgsConstructor;** we can use this

Reply:
added in [a4aa031](https://github.com/hmcts/rd-professional-api/pull/1529/commits/a4aa0314c9fdcfa34d663b7aa5778296934b4e2b)